core/kernel/tee_misc.c

```diff
@@ -118,3 +120,30 @@ bool core_is_buffer_intersect(paddr_t b, paddr_size_t bl,
 		return false;
 	return true;
 }
+
+void *tee_alloc_cache_aligned(size_t size)
```

Comment:
I suppose we can drop the `tee_` prefix.

Reply:
OK